HHS/Head-Start-TTADP
# [Prod] Updates to dev environment, add standard goals checkbox to ARs, correct national center name in historical data, fix bug with objective duplication

## Description of change
- Updates to the dev environment to speed up uninstalling dependencies and running a more lightweight local machine
- Add the "Use an OHS standard goals" checkbox to the AR
- Correct historical TR data to use the correct national center name
- Fix bug with objectives duplicating if they have disparate "objectiveCreatedHere" columns

## How to test
- The dev environment changes can only be tested in a local dev environment
- The bug with objectives duplicating requires a local environment as well to examine the specific support case
- To test the checkbox: add a goal to an AR using the "use an OHS standard goal" checkbox
- Confirm on a completed training report that the National Center is listed as PMFO.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2549
* https://jira.acf.gov/browse/TTAHUB-3483
* https://jira.acf.gov/browse/TTAHUB-3429
* https://jira.acf.gov/browse/TTAHUB-3528

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
